Makefile

```diff
@@ -153,7 +153,7 @@ test: generate gotestsum ## Run tests.
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
 
 .PHONY: test-integration
-test-integration: manifests generate envtest ginkgo mpi-operator-crd ray-operator-crd jobset-operator-crd ## Run tests.
+test-integration: manifests generate envtest ginkgo mpi-operator-crd ray-operator-crd jobset-operator-crd training-operator-crd ## Run tests.
```

Comment:
"training-operator" sounds a bit generic, in a way that can be confusing for future readers.
Maybe "tf-training-operator"?

Reply:
How about "kf-training-operator"?

Reply:
oops, that's what I wanted to say, sorry

Reply:
Done.